plugins/CoreAdminHome/API.php

```diff
@@ -139,7 +141,8 @@ public function setBrandingSettings($useCustomLogo)
      * @return array
      * @hideExceptForSuperUser
      */
-    public function invalidateArchivedReports($idSites, $dates, $period = false, $segment = false, $cascadeDown = false)
+    public function invalidateArchivedReports($idSites, $dates, $period = false, $segment = false, $cascadeDown = false,
+                                              $_forceInvalidateNonexistant = false)
```

Comment:
Explanation part 11: there are also some changes here to support invalidating ranges. `$dates` can be an array instead of a comma separated list, since range dates have commas. And `$_forceInvalidateNonexistant` is used in core:archive to make sure dummy range archives are created when normally we wouldn't want to for range archives.